src/req.rs

```diff
@@ -26,9 +26,17 @@ pub enum Version {
     V2,
 }
 
+#[derive(PartialEq, Eq, Debug, Clone, Deserialize, Serialize)]
+#[serde(untagged)]
+pub enum Id {
+    String(String),
```

Comment:
I want `String` to be wrapped by `Arc`, because `warp` does a lot of `clone` (if my understand is correct).
What do you think?

Reply:
Seems like you're correct, so I made the change in [168fea3](https://github.com/AtsukiTak/warp-json-rpc/pull/1/commits/168fea38303f283eaa0c411a9fc13fd4cdace82e).